### PR TITLE
Solves issue #210

### LINKED ISSUE #210: [Git] Update git push command with  "Force push with lease" flag
Update git push command [file](https://github.com/arshadkazmi42/ak-cli/blob/master/commands/git/git-push.md) with details about `--force-with-lease` flag

Ref: https://stackoverflow.com/a/52823955

### PR DESCRIPTION
**Pull request by arimallick**
Fixes #210
1. Introduced difference between git push --force & git push --force-with-lease
2. Fixed "GIF not showing" under DEMO tag in git-push.md

### CODE REVIEW

File: commands/git/git-push.md [hunk 1]
```diff
@@ -9,11 +9,20 @@
 
 `git push origin newbranch`
 
-### Demo
+#### Difference between:
+`git push --force` & `git push --force-with-lease`
+
+`force` overwrites a remote branch with your local branch. <br>
```
Comment (arshadkazmi42) [inline commands/git/git-push.md:15]:
```suggestion
`--force` overwrites a remote branch with your local branch. <br>
```

File: commands/git/git-push.md [hunk 2]
```diff
@@ -9,11 +9,20 @@
...
+
+`--force-with-lease` is a safer option that will not overwrite any work on the remote branch if more commits were added to the remote branch (by another team-member working on the same repository). It ensures you do not overwrite someone elses work by force pushing.
 
-<img src="../../gifs/git-push.gif" alt="Git Push"/> <br>
+[Click here to see the orginial git commit on the same.](https://github.com/git/git/commit/28f5d176110d2ed768a0a49159993c7a02d8cb15)
```
Comment (arshadkazmi42) [inline commands/git/git-push.md:19]: Let's remove this line. The main purpose of this repository is to simplify the details of the commands, this link provides a lot of information.

File: commands/git/git-push.md [hunk 3]
```diff
@@ -9,11 +9,20 @@
 
 `git push origin newbranch`
 
-### Demo
+#### Difference between:
```
Comment (arshadkazmi42) [inline commands/git/git-push.md:12]: Remove this title

File: commands/git/git-push.md [hunk 4]
```diff
@@ -9,11 +9,20 @@
...
+`git push --force` & `git push --force-with-lease`
```
Comment (arshadkazmi42) [inline commands/git/git-push.md:13]:
```suggestion
`git push --force` 

`git push --force-with-lease`
```

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Thank you for working on this.

I have added few comments can you work on those.

Once those are fixed we can get this merged

**Comment by arimallick** [comment]
I have made all the changes

**Review by arshadkazmi42 — APPROVED** [review]
Looks good. Thanks.

LGTM :tada:

**Pull request merged**